www/base/src/app/common/filters/limitStringLength.filter.js

```diff
@@ -0,0 +1,11 @@
+var limitStringLength = function ($filter) {
```

Comment:
I think this is indeed simpler, and does not add that much complexity.

@p12tic as you disagreed with this approach, I'd like your approval on this. I would understand your desire to  revert to previous design. (@slim-builder you would use 'git reflog' in order to find and go back to prev commit, with git reset)

Eventually, from a UX perspective, we should actually have a directive for that. And have that directive show the full-lenght string as a tooltip.

I think this goes outside of this PR.

Reply:
LGTM.